certification/pyxis/types.go

```diff
@@ -33,7 +33,7 @@ type CertImage struct {
 	RawConfig              string           `json:"raw_config,omitempty"`
 	Repositories           []Repository     `json:"repositories,omitempty"`
 	SumLayerSizeBytes      int64            `json:"sum_layer_size_bytes,omitempty"`
-	UncompressedTopLayerId string           `json:"uncompressed_top_layer_id,omitempty"` // TODO: figure out how to populate this, it is not required
+	UncompressedTopLayerID string           `json:"uncompressed_top_layer_id,omitempty"` // TODO: figure out how to populate this, it is not required
```

Comment:
We still need this Todo? Feel free to ignore

Reply:
No one has complained that this value is wrong, so I'd call it good to go, and the comment can be removed.

Reply:
I'll remove.